src/content/insertable-streams/video-processing/index.html

```diff
@@ -59,8 +59,10 @@ <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC
         <select id="transformSelector" disabled>
             <option selected value="webgl">WebGL</option>
             <option value="canvas2d">Canvas2D</option>
+            <option value="noop">Do nothing</option>
             <option value="drop">Drop frames at random</option>
```

Comment:
We might consider removing DropTransform in favor of NullTransform, since they don't really demonstrate unique uses of the API. (I just thought NullTransform would be too dull, so I tried to make it more interesting; however, NullTransform is more interesting for testing and experimentation.)

Reply:
When we get some kind of performance monitoring in there, NullTransform is interesting because it sets the lowest possible benchmark. DropTransform is kind of interesting in that it gives a jagged movement look, and it's one example of user-provided adaptation (lower bitrate by frame decimation). So I'd keep both.

(One of the things we can extend with when we pass through the PeerConnection is to measure the resulting bandwidth - DropTransform should show less bandwidth; the same should happen if we applied a WebGL "smoothing" filter).